schema/suricata.schema

```diff
@@ -0,0 +1,232 @@
+type suricata.common_ = record{
+  timestamp: time #time,
+  flow_id: count,
+  pcap_cnt: count,
+  src_ip: addr,
+  src_port: port,
+  dest_ip: addr,
+  dest_port: port,
+  proto: string,
+  community_id: string
+}
+
+type suricata.flow_ = record{
```

Comment:
What is the reason for the trailing `_` in the type name?

Reply:
It is only a component of a type, but not a full type that gets written by suricata. I'm thinking of renaming it to `suricata.component.flow`.